src/gatsby/sourceNodes/relayPiiNodes.ts

```diff
@@ -0,0 +1,22 @@
+import fs from 'fs';
+import path from 'path';
+
+export const piiFieldsNodes = ({ actions, createNodeId, createContentDigest }) => {
+    const { createNode } = actions;
+
+    const data = fs.readFileSync(path.join(__dirname, '../../data/relay_event_pii.txt'), 'utf-8');
```

Comment:
```suggestion
    const data = fs.readFileSync(path.join(__dirname, '..', '..', 'data', 'relay_event_pii.txt'), 'utf-8');
```

Just to be friendly to windows users checking out our docs locally.